querybook/server/lib/data_doc/dag_exporter/demo_dag_exporter.py

```diff
@@ -31,9 +59,41 @@ def dag_exporter_meta(self):
 
     def export(self, nodes, edges, meta):
         try:
+            export_dag = ""
+
+            for node in nodes:
+                node_section = '''
+    query = """{}"""
+    templated_query = dedent(query)
+
+    node{} = BashOperator(
```

Comment:
BashOperator? consider using sql operator like https://www.startdataengineering.com/post/how-to-schedule-a-sql-script-using-apache-airflow-with-an-example/